pina/label_tensor.py

```diff
@@ -66,7 +66,35 @@ def __init__(self, x, labels):
                 'the tensor has not the same number of columns of '
                 'the passed labels.'
             )
-        self.labels = labels
+        self._labels = labels
+
+    def _exist_labels(self):
+        """Check the existence of labels in a label tensor
+
+        :return: True if the tensor label exists, False otherwise
+        :rtype: bool
+        """
+        test = hasattr(self, 'labels')
+        return bool(test)
+
+    @property
+    def labels(self):
+        """Property decorator for labels
+
+        :return: labels of self
+        :rtype: list
+        """
+        return self._labels
+
+    @labels.setter
+    def labels(self, labels):
+        if not self._exist_labels():  # if labels dont not exist for self
```

Comment:
Here you are checking for existence of previously set labels. Only if they are not set, you check for length of labels and number of columns: basically for an already set LabelTensor, you can set whatever labels you want, which I think it's an error.

Reply:
Hi, I did not put the check on purpose. Consider the following 

```
        ....
        gradients.labels = input_.labels
        gradients = gradients.extract(d)
        gradients.labels = [f'd{output_fieldname}d{i}' for i in d]
```

which is done in [operator.py](https://github.com/mathLab/PINA/blob/d06f28de7b696588eb835002fe32d26bddd12d2c/pina/operators.py#L32), this would result in an error if I insert a check since you are modifying an already existing LabelTensor (changing labels for derivative).

I could put the check on LabelTensor and modify `operator.py` to not throw an error, what do you think?

Reply:
Yes!

Reply:
I made the modification as we discussed, everything should work now :)